mungegithub/mungers/approval-handler.go

```diff
@@ -340,9 +340,14 @@ func (h *ApprovalHandler) getMessage(obj *github.MungeObject, ownersMap map[stri
 // and identifies all of the people that have said /approve and adds
 // them to the approverSet.  The function uses the latest approve or cancel comment
 // to determine the Users intention
-func createApproverSet(comments []*githubapi.IssueComment) sets.String {
+func createApproverSet(obj *github.MungeObject, comments []*githubapi.IssueComment) sets.String {
 	approverSet := sets.NewString()
 
+	//prAuthor implicitly approves their own PR
+	if obj != nil && obj.Issue != nil && obj.Issue.User != nil && obj.Issue.User.Login != nil {
+		approverSet.Insert(*obj.Issue.User.Login)
+	}
+
```

Comment:
We agreed that author can't un-approve (because he's never approved). Is this code going to match this behavior?